validator/client/runner.go

```diff
@@ -83,24 +83,24 @@ func run(ctx context.Context, v Validator) {
 				wg.Add(1)
 				go func(role pb.ValidatorRole, id string) {
 					defer wg.Done()
+					tpk := id
+					if len(id) > 16 {
+						tpk = id[:16]
+					}
+					log := log.WithFields(logrus.Fields{
+						"pubKey": "0x" + tpk,
+						"role":   role,
+					})
 					switch role {
 					case pb.ValidatorRole_ATTESTER:
 						v.AttestToBlockHead(slotCtx, slot, id)
 					case pb.ValidatorRole_PROPOSER:
 						v.ProposeBlock(slotCtx, slot, id)
 						v.AttestToBlockHead(slotCtx, slot, id)
 					case pb.ValidatorRole_UNKNOWN:
-						pk12Char := id
-						if len(id) > 12 {
-							pk12Char = id[:12]
-						}
-						log.WithFields(logrus.Fields{
-							"public_key": pk12Char,
-							"slot":       slot,
-							"role":       role,
-						}).Debug("No active assignment, doing nothing")
+						log.Debug("No active role, doing nothing")
 					default:
-						// Do nothing :)
+						log.Warn("Unhandled role")
```

Comment:
Nice. This is one of those "this should never happen" paths, but a WARN is a good move, in case it does ever happen.